components/molecules/tabs/__tests__/stories.js

```diff
@@ -43,3 +45,17 @@ export const disabledStory = () => (
 disabledStory.story = {
```

Comment:
Do not need these as defaults to method name - sorry tried to explain this in chat earlier today.

```
BTW stories can be leaner

export const context = () =>
 <Table columns={columns} rows={dataContext} />

will take name from the method name
```